cmd/panfs.go

```diff
@@ -520,7 +514,13 @@ func (fs *PANFSObjects) MakeBucketWithLocation(ctx context.Context, bucket strin
 	}
 
 	// Create dir for temporary uploads
-	if err := mkdirAll(pathJoin(bucketMetaDir, tmpDir), 0o777); err != nil {
+	metaTmpPath := pathJoin(bucketMetaDir, tmpDir)
```

Comment:
What's the reason to change
`if err := mkdirAll(pathJoin(bucketMetaDir, tmpDir), 0o777); err != nil {`
to
```
metaTmpPath := pathJoin(bucketMetaDir, tmpDir)
if err := mkdirAll(metaTmpPath, 0o777); err != nil {
```
?
Lower you wrote the same construction as changed one:
`if err := mkdirAll(pathJoin(bucketMetaDir, mpartMetaPrefix), 0o777); err != nil {`

It would be better to write code with the identical logic in the identical style